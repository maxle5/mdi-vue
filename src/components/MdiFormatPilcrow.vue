
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M10,11A4,4 0 0,1 6,7A4,4 0 0,1 10,3H18V5H16V21H14V5H12V21H10V11Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiFormatPilcrow",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
