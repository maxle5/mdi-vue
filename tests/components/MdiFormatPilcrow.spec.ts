
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiFormatPilcrow from "../../src/components/MdiFormatPilcrow.vue";

test("MdiFormatPilcrow snapshot", () => {
  const wrapper = mount(MdiFormatPilcrow, {});
  expect(wrapper.html()).toMatchSnapshot();
});
