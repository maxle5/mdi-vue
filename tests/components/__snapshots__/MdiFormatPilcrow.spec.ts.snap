// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiFormatPilcrow snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M10,11A4,4 0 0,1 6,7A4,4 0 0,1 10,3H18V5H16V21H14V5H12V21H10V11Z" style="fill: currentColor;"></path>
</svg>"
`;
